src/ansys/mechanical/core/embedding/initializer.py

```diff
@@ -88,8 +88,18 @@ def _get_default_version() -> int:
     return int_version
 
 
+def _check_python_interpreter_architecture():
```

Comment:
If you want it to be private and not used anywhere else apart from this module, I'd go with a double leading underscore.